front/components/home/TrustedBy.tsx

```diff
@@ -5,25 +5,21 @@ import { classNames } from "@app/lib/utils";
 
 export default function TrustedBy() {
   return (
-    <>
-      <div
-        className={classNames(
-          "col-span-12 flex flex-col items-center py-8",
-          "lg:col-span-12 lg:col-start-1",
-          "xl:col-span-10 xl:col-start-2"
-        )}
-      >
-        <H3 className="w-full text-center text-white">
-          Trusted by 1000+ organizations
-        </H3>
-        <div
-          className={classNames(
-            "mt-8 w-full",
-            "grid grid-cols-2 place-items-center gap-1",
-            "sm:grid-cols-3 sm:gap-0",
-            "md:grid-cols-4 md:gap-0"
-          )}
-        >
+    <div
+      className={classNames(
+        "col-span-12 flex flex-col items-center py-4",
+        "lg:col-span-12 lg:col-start-1",
+        "xl:col-span-10 xl:col-start-2"
+      )}
+    >
+      <H3 className="w-full text-center text-white">
+        Trusted by 1000+ organizations
+      </H3>
+
+      <div className="mx-auto mt-8 w-full max-w-[1600px] px-8">
```

Comment:
Let's try to avoid having pixel values, let's use [tailwind classes](https://tailwindcss.com/docs/max-width)